test/unit/lib/plugins/aws/rollback.test.js

```diff
@@ -123,8 +123,8 @@ describe('AwsRollback', () => {
         .then(() => {
           assert.isNotOk(true, 'setStackToUpdate should not resolve');
         })
-        .catch((errorMessage) => {
-          expect(errorMessage).to.include("Couldn't find any existing deployments");
+        .catch((error) => {
+          expect(error.message).to.include("Couldn't find any existing deployments");
```

Comment:
When switching to `ServerlessError`, we also confirm that expected error happened, just by confirming on its `code` (we do not confirm on `message` as that's subject to grammar improvements and those should not break tests)